src/gleam/list.gleam

```diff
@@ -1565,6 +1565,54 @@ pub fn each(list: List(a), f: fn(a) -> b) -> Nil {
   }
 }
 
+/// Used to iterate through a list of items and execute a side effect per item as long as the
+/// previous side effect execution returned an `Ok(_)`.
+///
+/// Takes a function that returns a `Result`. The function is applied to each list item.
+/// If the function application on an item returns `Ok(_)`, then `try_each` will go on and apply the
+/// function to the next item.
+///
+/// However, if any application returns an `Error(_)`, at that point the function will stop executing.
+/// In any case the function returns `Nil`, even if it passes through all items or incurs in an
+/// `Error`.
+///
+/// ## Examples
+///
+/// ```gleam
+/// > try_each(
+/// >   over: [1, 2, 3],
+/// >   with: fn(x) {
+/// >     io.print(int.to_string(x)) // prints "1" "2" "3" as single side effects.
```

Comment:
I prefer `int.to_string(x) |> io.print()`

Reply:
Ok! Though I'd prefer `int.to_string(x) |> io.print`, is there a reason to prefer to put parentheses on `io.print()`?